src/app/body/quick-notes/quick-notes.component.ts

```diff
@@ -72,6 +72,13 @@ export class QuickNotesComponent implements OnInit {
   }
 
   deleteNote(docId: string) {
+    for(let i=0; i<this.quickNotes.notes.length; i++){
+      console.log(this.quickNotes.notes[i].DocId)
+      
+      if(this.quickNotes.notes[i].DocId==docId){ 
+      delete this.quickNotes.notes[i]; 
```

Comment:
We don't need to completely delete the note, we're just marking it as deleted for now. So we don't need this.